### PR TITLE
Move project page tests

### PR DESCRIPTION
**Pull request by BryceLTaylor**
### Resolves:

Fixes the skipped project page tests in legacy integration tests.

### Changes:

Moves the project page integration tests from Tap to Jest and makes sure they pass.  Also changes how tests are run so that you must supply a project id as an environment variable to run tests against.

The testing section of the README has been updated to reflect the changes to how tests are run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,11 +215,13 @@ so for the time being our tests run using both.
 To run all integration tests from the command-line:
 
 ```bash
-SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu npm run test:integration
+SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu PROJECT_ID=id# npm run test:integration
```
Comment (rschamp) [inline README.md:218]: I think we should keep this copy-pasteable

```suggestion
SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu TEST_PROJECT_ID=1 npm run test:integration
```

File: test/integration/project-page.test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const {
+    findByXpath,
+    clickXpath,
+    buildDriver
+} = new SeleniumHelper();
+
+let remote = process.env.SMOKE_REMOTE || false;
+let rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
+let projectId = process.env.TEST_PROJECT_ID;
```
Comment (rschamp) [inline test/integration/project-page.test.js:11]:
```suggestion
let projectId = process.env.TEST_PROJECT_ID || 1300006196;
```

File: test/integration/project-page.test.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    });
+
+    beforeEach(async () => {
+        await driver.get(projectUrl);
+        let gfOverlay = await findByXpath('//div[@class="stage-wrapper_stage-wrapper_2bejr box_box_2jjDp"]');
+        await gfOverlay.isDisplayed();
+    });
+
+    afterAll(async () => await driver.quit());
+
```
Comment (rschamp) [inline test/integration/project-page.test.js:36]:
```suggestion

    // LOGGED OUT TESTS
    
```

**Review by rschamp — CHANGES_REQUESTED** [review]

File: test/integration/project-page.test.js (hunk 3 above)
Comment (BryceLTaylor) [reply test/integration/project-page.test.js:36]: Added

File: test/integration/project-page.test.js (hunk 2 above)
Comment (BryceLTaylor) [reply test/integration/project-page.test.js]: Added

File: README.md (hunk 1 above)
Comment (BryceLTaylor) [reply README.md]: Sounds good

**Review by rschamp — APPROVED** [review]

**Pull request merged**